eslint/eslintrc.yml

```diff
@@ -23,15 +23,21 @@ globals:
 
 extends:
     # Extend the base Body Labs style, which is pulled in via peerDependencies.
-    "../../bodylabs-javascript-style/eslint/eslintrc-shared.yml"
+    - '../../bodylabs-javascript-style/eslint/eslintrc-shared.yml'
 
-rules:
-    # 2 means error, 1 means warning, 0 means ignore.
+    # Pull in https://github.com/yannickcr/eslint-plugin-react
+    - 'plugin:react/recommended'
 
+rules:
     # JSX.
-    # eslint-plugin-react is required
-    # https://github.com/yannickcr/eslint-plugin-react
-    jsx-quotes: 2
+    jsx-quotes: 'error'
+
+    # More JSX, from https://github.com/yannickcr/eslint-plugin-react
+    react/jsx-curly-spacing: ['error', 'always']
+    react/jsx-equals-spacing: ['error', 'never']
+    react/jsx-indent: 'error'
+    react/jsx-pascal-case: 'error'
+    react/jsx-space-before-closing: 'error'
 
```

Comment:
There's a few other rules available that we could use that seem like no-brainers to me: Out of the [JSX-specific rules](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules) pretty much all of them seem like good ideas, but I'm especially for jsx-key, jsx-no-duplicate-props, and jsx-sort-props.